projects/nuvoton/numaker_iot_m487_wifi/uvision/aws_demos/aws_demos.uvproj

```diff
@@ -374,7 +374,7 @@
 							<MiscControls>--diag_suppress=550,177,C4017,111,2770,223</MiscControls>
```

Comment:
Please also update the nuvoton ethernet file projects\nuvoton\numaker_iot_m487_wifi\uvision\aws_demos_eth\aws_demos_eth.uvproj

You can just add another file xml element under where mqtt_demo_plaintext.c is:
````
						<File>
							<FileName>mqtt_demo_mutual_auth.c</FileName>
							<FileType>1</FileType>
							<FilePath>../../../../../demos/coreMQTT/mqtt_demo_mutual_auth.c</FilePath>
						</File>
```

All of the include path and files for retry utils and transport secure sockets is already there.